lib/common/logging.c

```diff
@@ -798,6 +796,7 @@ set_identity(const char *entity, int argc, char *const *argv)
 
     CRM_ASSERT(crm_system_name != NULL);
 
+    // @COMPAT Drop at 3.0.0. Added via f3c4f9e without explanation. Unused.
     setenv("PCMK_service", crm_system_name, 1);
```

Comment:
See fence-agents/lib/fencing.py.py

Reply:
wow

wish we had a compendium of ClusterLabs users of obscure Pacemaker features